script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt

```diff
@@ -337,39 +351,40 @@ It seems like this time she cooperated since your interests aligned, but you mus
 [sha:09dd7d48e3835955b7e66d579b465acf20700843]{
 -- Page 216, Offset 15313.
 -- 先輩の言葉に、俺は素直に頷けなかった。
-I cannot so readily accept Senpai's words.
+I can't accept Senpai's words that easily.
 }
 [sha:f937e967c0fe5f0428654e695bba71b508229c08]{
 -- Page 216, Offset 15314.
 -- きっと先輩が正しい。アルクェイドが吸血鬼である事はどうしようもない事実だ。
-// even if added her to flow better with the previous line.
-Even if, in all likelihood, she's right. Arcueid being a vampire is an undeniable fact.
+There's a good chance what she's saying is true.
```

Comment:
I don't really like this change, it feels like his level of belief got reduced too much. I read it as more like "I'm sure she's right (but I don't wanna believe it)"